javascript/javascript2/week3/FuncsAsVars.js

```diff
@@ -0,0 +1,30 @@
+// Create an array with 3 items. All items should be functions. Iterate through the array and call all the functions.
+const funcArray = [funcA(), funcB(), funcC()];
+function funcA() {
+  console.log("a");
+}
+function funcB() {
+  console.log("b");
+}
+function funcC() {
+  console.log("c");
+}
+for (let i = 0; i < funcArray.length; i++) {
+  funcArray[i];
+}
```

Comment:
It seems that you are calling the functions when you are creating the array, and not when you are iterating through them. Try to only add the functions to the array without calling them, and then to call them when you iterate through the array :)

Reply:
Yes, it was a mistake. Thank you
I've fixed this